tyrant/constants.py

```diff
@@ -120,6 +120,11 @@
     "To power a flashlight bulb, you need 500 wired lemons to conduct electricity.",
     "The heaviest lemon was 11 pounds, 9.7 ounces in 2003.",
     "Lemon was a common unisex name in the 1900’s.",
+    "Lemons are nature’s best insect repellent.",
+    "The name “lemon” may have originated from the Middle East",
```

Comment:
```suggestion
    "The name 'lemon' may have originated from the Middle East.",
```
Missing full stop, uses non-standard quotes around `lemon`